x-pack/plugins/observability_solution/profiling/public/plugin.tsx

```diff
@@ -64,29 +65,47 @@ export class ProfilingPlugin
     ];
 
     const kuerySubject = new BehaviorSubject<string>('');
+    const appUpdater$ = new BehaviorSubject<AppUpdater>(() => ({}));
 
     const section$ = combineLatest([from(coreSetup.getStartServices()), kuerySubject]).pipe(
       map(([[coreStart], kuery]) => {
         if (coreStart.application.capabilities.profiling.show) {
-          const sections: NavigationSection[] = [
-            {
-              label: i18n.translate('xpack.profiling.navigation.sectionLabel', {
-                defaultMessage: 'Universal Profiling',
-              }),
-              entries: links.map((link) => {
-                return {
-                  app: OBLT_PROFILING_APP_ID,
-                  label: link.title,
-                  path: `${link.path}?kuery=${kuery ?? ''}`,
-                  matchPath: (path) => {
-                    return path.startsWith(link.path);
-                  },
-                };
-              }),
-              sortKey: 700,
-            },
-          ];
-          return sections;
+          let isSidebarEnabled = true;
+          coreStart.chrome
```

Comment:
It's definitely more readable when helper function is used. I hesitated to use it because I wasn't sure if it handles unsubscribing outside of React component, but when checking the source code, I see that it suppose to work. I'll more forward with the change.

Reply:
Oh, I see that `useObservable` can't be used outside of React components. It throws an error :/
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/5f09e6d9-0398-4074-816c-39d9a6af7d00">

Reply:
I didn't catch it! 

Renamed `plugin.tsx` to `plugin.ts` alongside with `index.tsx` to `index.ts`.

Thank you :)